src/NDItk/multigroup/Metadata.hpp

```diff
@@ -0,0 +1,159 @@
+#ifndef NJOY_NDITK_MULTIGROUP_METADATA
+#define NJOY_NDITK_MULTIGROUP_METADATA
+
+// system includes
+
+// other includes
+#include "tools/Log.hpp"
+#include "NDItk/base/SingleIntegerRecord.hpp"
+#include "NDItk/base/SingleRealRecord.hpp"
+#include "NDItk/base/SingleStringRecord.hpp"
+
+namespace njoy {
+namespace NDItk {
+namespace multigroup {
+
+/**
+ *  @brief Metadata associated to a neutron NDI file
```

Comment:
Consider it changed (I'll push a commit with changes later).

As a side note: we should not shy away from adding more rather than less documentation. If you think we can elaborate, we can add more documentation in a future PR.